store/localcachelayer/user_layer.go

```diff
@@ -39,6 +49,48 @@ func (s LocalCacheUserStore) InvalidatProfileCacheForUser(userId string) {
 	}
 }
 
+func (s LocalCacheUserStore) InvalidateProfilesInChannelCacheByUser(userId string) {
```

Comment:
This function and the next one are used in the `app/web_hub.go` file, at the same time that same file is sending the invalidation event to the cluster, now all that cluster invalidation logic is handled by this layer, so you have to update the code in `app/web_hub.go` to not sent the cluster message for this 2 cases.